src/coreclr/vm/jitinterface.cpp

```diff
@@ -4634,6 +4634,25 @@ bool CEEInfo::isExactType(CORINFO_CLASS_HANDLE cls)
     return result;
 }
 
+// Returns true if a class handle represents a generic type.
+bool CEEInfo::isExactType(CORINFO_CLASS_HANDLE cls)
```

Comment:
Where is this defined? Should this be `isGenericType()`?

Reply:
Yes. But you see, I'm really bad at remembering to change names after copy pasting stuff 😆